person.rb

```diff
@@ -1,11 +1,14 @@
 require './nameable'
 class Person < Nameable
+  attr_reader :rentals
+
   def initialize(age, name = 'unknown', parent_permission: true)
     super()
     @name = name
     @id = Random.rand(1..100)
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] Kindly make sure to add a method for adding rentals in the `Person` class
e.g
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
 end
```